StructureDefinition/Extension-NHSDigital-Provenance-agent.StructureDefinition.xml

```diff
@@ -95,7 +95,6 @@
         <severity value="error" />
         <human value="SDS Role Profile Id must be 12 digits" />
         <expression value="identifier.where(system='https://fhir.nhs.uk/Id/sds-role-profile-id').exists().not() or (identifier.where(system='https://fhir.nhs.uk/Id/sds-role-profile-id').exists()  and identifier.where(system='https://fhir.nhs.uk/Id/sds-role-profile-id').value.matches('^[0-9]{12}$'))" />
-        <source value="https://fhir.nhs.uk/StructureDefinition/NHSDigital-PractitionerRole" />
```

Comment:
Removed sources as constraints cannot be imported from other structuredefinitions, only inherited, in which case they do not appear in the SD. removal of the source means this constraint is defined as a new constraint for the profile. This change would not have an impact on validation.